generators/entity/index.js

```diff
@@ -223,6 +223,18 @@ module.exports = class extends BaseGenerator {
                     this.log(chalk.green(`\nFound the ${context.filename} configuration file, entity can be automatically generated!\n`));
                     context.useConfigurationFile = true;
                 }
+
+                context.entitySuffix = configuration.get('entitySuffix');
+                if (context.entitySuffix === undefined || context.entitySuffix === null) context.entitySuffix = '';
```

Comment:
```javascript
if (!context.entitySuffix) {
   context.entitySuffix = '';
}
```

Reply:
Same as my previous comment, I need to do this : 
```javascript
if (_.isNil(context.dtoSuffix)) {
   context.dtoSuffix = 'DTO';
}
```